src/layouts/default.layout.scss

```diff
@@ -5,18 +5,23 @@
 @use '../styles/themes/light';
 @use '../styles/themes/dark';
 
+@use '../styles/colors';
+
 @include sourceSans.fontFace($weight: 400, $fontDir: '~/@fontsource/source-sans-pro/files');
 @include sourceSans.fontFace($weight: 600, $fontDir: '~/@fontsource/source-sans-pro/files');
 
 :root {
-  @include base.theme;
-  
-  // default theme
+  @include colors.colors;
+
+  @include base.theme;  
+}
+
+:root[theme="light"] {
   @include light.theme;
+}
 
-  @media(prefers-color-scheme: dark) {
```

Comment:
I'd tend to say, that we should stick to the `prefers-color-scheme` as long as no theme has been explicitly set. Thus, the theme adopts to the users preferences (e.g. dynamically change scheme at runtime by the os). Once the theme is explicitly set, the attribute selector overrides this.

To keep it simple we would not offer an option to opt-out of an explicitly selected theme, unless you reload the page (or simply navigate).

So either way, this should be _opt-in_ and we should later on think about persisting this setting (e.g. in `sessioStorage` or `localStorage` or ones moms cookies...)